driver-core/src/test/java/com/datastax/driver/core/querybuilder/QueryBuilderTest.java

```diff
@@ -300,6 +300,24 @@ public void updateTest() throws Exception {
     }
 
     @Test(groups = "unit")
+    public void should_handle_update_if_exists() {
```

Comment:
Well actually if we're getting a SyntaxException, it means the CQL grammar does not accept it, so it's very unlikely that one day it would ever be accepted.